storage/base.h

```diff
@@ -156,12 +156,12 @@ struct TypeListMapperImpl;
 template <typename FIELDS, int... NS>
 struct TypeListMapperImpl<FIELDS, current::variadic_indexes::indexes<NS...>> {
 #ifdef CURRENT_STORAGE_PATCH_SUPPORT
-  using result = TypeList<typename std::result_of<FIELDS(FieldInfoByIndex<NS>)>::type::update_event_t...,
-                          typename std::result_of<FIELDS(FieldInfoByIndex<NS>)>::type::delete_event_t...,
-                          typename std::result_of<FIELDS(FieldInfoByIndex<NS>)>::type::patch_event_t...>;
+  using result = TypeList<typename std::result_of_t<FIELDS(FieldInfoByIndex<NS>)>::update_event_t...,
+                          typename std::result_of_t<FIELDS(FieldInfoByIndex<NS>)>::delete_event_t...,
+                          typename std::result_of_t<FIELDS(FieldInfoByIndex<NS>)>::patch_event_t...>;
```

Comment:
@dkorolev Is `typename` required here?

Reply:
Yep, it's for the latter `{update,delete,patch}_event_t`. I checked :-)